client/app/bundles/KudosApp/components/Kudo.jsx

```diff
@@ -1,5 +1,5 @@
 import React, { PropTypes } from 'react';
-import _ from 'lodash';
+import _, { map } from 'lodash';
```

Comment:
We should probably be consistent throughout this project. E.g. either just use `_.map` in the code, or we could go through the other .jsx files and change the imports to be explicit for the particular lodash function 
`import { bindAll, forEach, map } from 'lodash';`

Reply:
Agree ^.

We should decide on a standard and at minimum change as we touch files (so no need to go through the entire codebase at this point, but at least this file should be consistent).

Reply:
Sounds good!  Leave the campground cleaner than you found it.